tests/test_opds_import.py

```diff
@@ -286,59 +296,135 @@ def test_import(self):
         classifier = Classifier.classifiers.get(seven.subject.type, None)
         classifier.classify(seven.subject)
 
+        # If we import the same file again, we get the same list of Editions.
+        imported_editions_2, imported_pools_2, imported_works_2, error_messages_2, next_links_2 = (
+            OPDSImporter(self._db).import_from_feed(feed)
+        )
+        eq_(imported_editions_2, imported_editions)
+
+        # importing with a lendable data source makes license pools and works
+        imported_editions, imported_pools, imported_works, error_messages, next_links = (
+            OPDSImporter(self._db, data_source_name=DataSource.OA_CONTENT_SERVER).import_from_feed(feed)
+        )
+
+        [crow, mouse] = sorted(imported_editions, key=lambda x: x.title)
+
+        # Work was created for both books.
+        assert crow.license_pool.work is not None
+        eq_(Edition.BOOK_MEDIUM, crow.medium)
+
+        # But the 'mouse' book is known to come from Project Gutenberg,
+        # so a Work has been created for that book.
+        assert mouse.license_pool.work is not None
+        eq_(Edition.PERIODICAL_MEDIUM, mouse.medium)
+
         work = mouse.license_pool.work
         work.calculate_presentation()
         eq_(0.4142, round(work.quality, 4))
         eq_(Classifier.AUDIENCE_CHILDREN, work.audience)
         eq_(NumericRange(7,7, '[]'), work.target_age)
 
-        # The other book has no license pool and no work because we
-        # could not figure out whether the license source was Project
-        # Gutenberg or Project GITenberg.
-        eq_(None, crow.work)
-        eq_(None, crow.license_pool)
-
         # Bonus: make sure that delivery mechanisms are set appropriately.
         [mech] = mouse.license_pool.delivery_mechanisms
         eq_(Representation.EPUB_MEDIA_TYPE, mech.delivery_mechanism.content_type)
         eq_(DeliveryMechanism.NO_DRM, mech.delivery_mechanism.drm_scheme)
         eq_('http://www.gutenberg.org/ebooks/10441.epub.images', 
             mech.resource.url)
 
-        # If we import the same file again, we get the same list of Editions.
-        imported2, messages, next_links = OPDSImporter(self._db).import_from_feed(feed)
-        eq_(imported2, imported)
+
+
+    def test_import_with_wrangler_data_source(self):
+        # Tests that will create Edition, LicensePool, and Work objects, when appropriate.
+        # For example, on a Metadata_Wrangler data source, it is only appropriate to create 
+        # editions, but not pools or works.  On a lendable data source, should create 
+        # pools and works as well as editions.
```

Comment:
this seems to duplicate some of the previous test - the cutoff is the only new thing that's being tested here